types.go

```diff
@@ -610,10 +616,32 @@ func (a Asset) MarshalJSON() (data []byte, err error) {
 	return json.Marshal(a.String())
 }
 
+// added 2022-11-02
```

Comment:
Use standard Go comment structure here which should start with the struct's name so like:

```
// RexInfo was added on version <>
```

Reply:
https://github.com/eoscanada/eos-go/pull/191/commits/18d78b47dd7be8c506ce92f5422a4273eb0d1820